src/index.js

```diff
@@ -3,7 +3,18 @@ import ReactDOM from 'react-dom';
 import 'typeface-roboto';
 import './index.css';
 import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import thunk from 'redux-thunk';
+import {Provider} from 'react-redux';
+import rootReducer from './redux/rootReducer';
+import {applyMiddleware, compose, createStore} from 'redux';
```

Comment:
Wie wollen wir die Imports organisieren? Zufällige Reihenfolge oder eine Logik wie z.B. zuerst Frameworks/Libraries und dann die Komponenten in alphabetischer Reihenfolge?

Reply:
Ich glaube bei mir ist es so, dass IntelliJ die Reihenfolge macht nach dem formatieren. Aber ich bin offen, falls wir eine andere Reihenfolge verwenden wollen

Reply:
Schauen wir, wie es funktionieren wird :-)